### DISCUSSION TITLE
format @clip by making the text lowercase

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#129] [Q&A] [answered]
![image](https://github.com/buggyj/tiddlyclip-plugin/assets/67494083/516d82d7-aa7e-4cc2-818e-7abf4e05a578)

This is the rule I use to create tiddler titles for clippings from twitter.
I want to configure @clip part such that keep the copied text is completely lowercase. 
For example, if the @clip is `Cystic fibrosis Liver disease`, I want it to be formatted as `cystic-fibrosis-liver-disease`

Is that possible ?

**Comment by buggyj** [comment, marked as answer]
add this tid -

use

`@clip=((*@toLowerCase(@clip)*))`

[$ _plugins_bj_tiddlyclip_lowcase.js.json](https://github.com/buggyj/tiddlyclip-plugin/files/14584601/_plugins_bj_tiddlyclip_lowcase.js.json)

**Reply by arunnbabu81** [reply]
Thanks @buggyj
